### ISSUE TITLE
Ensure screen readers automatically focus on the header of the screen

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#562]
On the Credential Offer screen, the first thing the screen reader reads out is “go back button”. It should read "credential offer".

When the camera opens to scan a QR code, the screen reader reads out “BC Wallet close” (which is the exit button label) as the first and only information. It should read "Scanning".

**Comment by cvarjao** [comment]
Need some work to figure out the how screen readers interprets the order of elements

**Comment by knguyenBC** [comment]
Read WCAG guidance on [focus order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order) and [meaningful sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html#:~:text=A%20sequence%20is%20meaningful%20if,as%20they%20are%20not%20interleaved.). It seems like its up to us to determine if the back button being read before the screen header would be unexpected or would change the meaning of the content of the screen. I checked other apps and none of the ones I checked read the header before the back button, so this behaviour may be expected from users. I also do not think the location of the back button changes the meaning of the screen. I think we can close this ticket.
